src/tokens/colour/base.json

```diff
@@ -0,0 +1,105 @@
+{
+  "colour": {    
+    "black":     { "value": "#000" },
+    "black-rgb": { "value": "0, 0, 0" },
+    "white":     { "value": "#fff" },
+    "white-rgb": { "value": "255, 255, 255" },
```

Comment:
I'm wondering whether we couldn't auto-generate the rgb variants, rather than specify them manually ?

Reply:
I like that idea. We could also auto output different formats such as HSL in future. It would also negate the [issues around key names](https://github.com/wellcometrust/design-system/pull/2#discussion_r668513198) you highlighted

Reply:
RGB values removed from the token file in favour of automatically inserting using a Style Dictionary transform.